src/Microsoft.AspNetCore.Diagnostics.HealthChecks/HealthCheckAppBuilderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static class HealthCheckAppBuilderExtensions
+    {
+        public static IApplicationBuilder UseHealthChecks(this IApplicationBuilder applicationBuilder, PathString path)
```

Comment:
Overload that accepts options func?

Reply:
No options right now. But as we add some, sure.

Reply:
Aren't Options all supposed to be configured as services now? Most Use methods no longer take parameters.